README.md

```diff
@@ -1,50 +1,41 @@
-# About
```

Comment:
Can you undo your changes to the README?  You removed the link to the software license, docker instructions, and other important info.  

Instead, I think you just need to add your Step 5 instruction about the bootstrap commands to the following installation guide file: https://github.com/spacemunkay/BikeShed/blob/master/doc/local_dev_setup.md